### PR TITLE
dotnet test CLI ref updates

### LINKED ISSUE #17135: NUnit filter example is not covered
In the section with list of example it is covered only _MSTest_ and _xUnit_.   
I'd like to see also _NUnit_.  
For example I don't know how to filter tests with the **CategoryAttribute**.  
Should I use **Category**=mycategory or **Trait** or **Traits** ?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 61ca4064-7481-a263-60fc-c70127f2ad16
* Version Independent ID: 2878af44-8208-7bd8-46d4-3acda3e82ce0
* Content: [dotnet test command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test?tabs=netcore21)
* Content Source: [docs/core/tools/dotnet-test.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-test.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### LINKED ISSUE #17256: The --collect option should include code coverage example
https://github.com/microsoft/vstest/issues/981

Related to #17255

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 61ca4064-7481-a263-60fc-c70127f2ad16
* Version Independent ID: 2878af44-8208-7bd8-46d4-3acda3e82ce0
* Content: [dotnet test command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test)
* Content Source: [docs/core/tools/dotnet-test.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-test.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #17256
Fixes #17135

### CODE REVIEW

**Comment by tdykstra** [comment]
@nohwnd please review

**Comment by nohwnd** [comment]
@tdykstra @jakubch1 would be a better person to review this, because he is focusing on code coverage. 

@jakubch1 please review this.

File: docs/core/tools/dotnet-test.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ Test projects specify the test runner using an ordinary `<PackageReference>` ele
 - **`--collect <DATA_COLLECTOR_FRIENDLY_NAME>`**
 
   Enables data collector for the test run. For more information, see [Monitor and analyze test run](https://aka.ms/vstest-collect).
+  
+  To collect code coverage on any platform that is supported by .NET Core, install [Coverlet](https://github.com/tonerdo/coverlet/blob/master/README.md) and use the `--collect:"XPlat Code Coverage"` option.
```
Comment (jakubch1) [inline docs/core/tools/dotnet-test.md:68]: let's use new link to coverlet:
https://github.com/coverlet-coverage/coverlet/blob/master/README.md

**Review by jakubch1 — APPROVED** [review]

File: docs/core/tools/dotnet-test.md (hunk 1 above)
Comment (nohwnd) [reply docs/core/tools/dotnet-test.md:68]:
```suggestion
  To collect code coverage on any platform that is supported by .NET Core, install [Coverlet](https://github.com/coverlet-coverage/coverlet/blob/master/README.md) and use the `--collect:"XPlat Code Coverage"` option.
```

**Comment by tdykstra** [comment]
@jakubch1 @nohwnd Thanks for review!

**Review by BillWagner — APPROVED** [review]
This LGTM @tdykstra 

You can :shipit: when ready.

**Pull request merged**